docs/terminal/getting-started.md

```diff
@@ -0,0 +1,156 @@
+
+1. Select one of the links to open the file in the editor at that specific line in the file.
+
+    ![Navigate to a specific line in a file](./images/getting-started/terminal-line-column.png)
+
+Learn more about the different types of [links in the terminal](/docs/terminal/basics.md#links).
+
+## Navigate to previous commands
+
+As you work in the terminal, you might need to review a previous command and its output, or maybe you want to rerun a command. You can quickly navigate to the previous commands by using keyboard shortcuts.
+
+To navigate to previous commands:
+
+1. Open the terminal you used previously.
+
+1. Press the `kb(workbench.action.terminal.scrollToPreviousCommand)` keyboard shortcut to scroll to the previous command in the terminal history.
+
+    Notice that the terminal scrolls to the previous command and highlights it.
+
+    ![Navigate to the previous command](./images/getting-started/previous-command.png)
+
+    If you press `kb(workbench.action.terminal.scrollToPreviousCommand)` multiple times, the terminal scrolls further through the command history. You can use the `kb(workbench.action.terminal.scrollToNextCommand)` keyboard shortcut to navigate in the other direction.
+
+1. You might see a circle icon in the gutter next to a previously run command. Select the circle icon, and then select **Rerun Command** to run the command again.
+
+    ![Rerun a command](./images/getting-started/rerun-command.png)
+
+Learn more about [navigating through the command history](/docs/terminal/shell-integration.md#command-navigation).
+
+## Run commands in another shell
+
+The integrated terminal supports having multiple terminals open at the same time. For example, you could dedicate one terminal to run Git commands and another terminal to run build scripts. You can also run commands in different shells based on your preference.
+
+To add a new terminal in a different shell:
+
+1. Select the `˅` icon in the terminal panel to open the terminal dropdown, and then select from one of the available shells.
+
+    > **Note**: The available shells depend on the shells installed on your machine.
+
+    ![Select a different shell](./images/getting-started/select-shell.png)
+
+    A new terminal opens with the selected shell, where you can enter commands like you did previously.
+
+    > **Tip**: You can also select the `+` icon to create a new terminal for the default shell, use the `kb(workbench.action.terminal.new)` keyboard shortcut, or select **Terminal** > **New Terminal** from the menu bar.
+
+1. You can view the open terminals from the list in the terminal panel.
+
+    ![View the list of terminals](./images/getting-started/terminal-list.png)
+
+    To switch to another terminal, select it from the list of terminals.
+
+    > **Tip**: You can rename a terminal in the list by right-clicking on it and selecting **Rename**.
+
+1. Right-click a terminal in the terminal list and select **Move Terminal into Editor Area**.
```

Comment:
Dragging is the most intuitive method for this, should we just be mentioning that approach instead of the context menu entry?